extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -952,6 +957,47 @@ export default class TypeScriptServiceClient extends Disposable implements IType
 			this.executeWithoutWaitingForResponse('configurePlugin', { pluginName, configuration });
 		}
 	}
+
+	private notifyExtendedLogging(context: vscode.ExtensionContext) {
+		const getLastChange = (): Date | undefined => {
```

Comment:
There needs to be a way to disable this prompt too. I always keep this setting enabled so I can easily collect logs when something weird happens